eclass/systemd.eclass

```diff
@@ -98,7 +98,7 @@ systemd_get_unitdir() {
 systemd_get_userunitdir() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	_systemd_get_dir systemduserunitdir /usr/lib/systemd/user
+	_systemd_get_dir systemduserunitdir /lib/systemd/user
```

Comment:
This is incorrect.

```
$ pkg-config --variable=systemduserunitdir systemd
/usr/lib/systemd/user
```

Reply:
I raise you to: 

```
pkg-config --variable=systemdsystemunitdir systemd
/usr/lib/systemd/system

🐧 pkg-config --variable=systemdutildir systemd
/usr/lib/systemd

🐧 pkg-config --variable=systemdsystemgeneratordir systemd
/usr/lib/systemd/system-generators

🐧 pkg-config --variable=systemdsystempresetdir systemd
/usr/lib/systemd/system-preset

🐧 pkg-config --variable=systemdsleepdir systemd
/usr/lib/systemd/system-sleep
```

So should I change all /lib fallback paths with /usr/lib?

Reply:
That's not how vanilla systemd install on Gentoo behaves.

Reply:
The system paths var depending on the split-usr USE flag. We can switch the defaults from /lib to /usr/lib after we drop split-usr support from systemd.